boards/arduino/opta/arduino_opta-common.dtsi

```diff
@@ -58,6 +58,14 @@
 		};
 	};
 
+	sdram2: sdram@d0000000 {
```

Comment:
Unluckily, Opta doesn't have 32MB of SDRAM :sweat_smile: 
Please remove the entry.

Reply:
My fault, I was too fast with a copy&paste when I setup the eth part and then never asked myself why that entry was there. Removed.